src/prepare/PrepareBase.ts

```diff
@@ -37,8 +37,7 @@ export abstract class PrepareBase
     protected timeout?: number;
 
     /**
-     * * @param {Renderer} renderer - A reference to the current renderer
-     * @param renderer
+     * @param {Renderer} renderer - A reference to the current renderer
```

Comment:
Should the type here be `rendering.Renderer`?

Reply:
You are right! I've pushed this change to this PR